### PR TITLE
New Minecraft PE kb and images

### PR DESCRIPTION
**Pull request by BobStolzberg**
New Minecraft PE kb.  This is the server that iOS clients connect to.
Images are screen shots.

### CODE REVIEW

File: Ecosystem Partners/getting-started-with-minecraft-pe-server-blueprint.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  1. Server Name - Server Name, e.g. CLCMC
+  2. Server Description - Description of your server, e.g. Family Minecraft Server
+  3. Game Mode - Creative or Survival mode
+  4. minecraftpe user's password - Set a secure password for the minecraftpe user that runs the server software.
+  5. Click “next: step 2”
+  ![Configure Minecraft PE Blueprint Image](../images/ecosystem-minecraft-pe-server-11.png)
+
+4. Review, Confirm and Deploy the Blueprint.
+  1. Verify your configuration details.
+  2. Once verified, click on the ‘deploy blueprint’ button. This will kick off the blueprint deploy process and load a page to allow you to track the progress of the deployment.
```
Comment (osbornm) [inline Ecosystem Partners/getting-started-with-minecraft-pe-server-blueprint.md]: Please review all the single and double quotes and make sure they are not `straight quotes` editor like Microsoft Word convert them to a unicode character that show forward and backward direction. 

http://www.techrepublic.com/forums/questions/how-to-force-ms-word-to-use-plain-single-and-double-quotes/

File: Ecosystem Partners/getting-started-with-minecraft-pe-server-blueprint.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+1. Add a user called minecraftpe to the Linux OS.  This user will run the Minecraft PE server software so the inecraft PE Server will not run as root.  The minecraftpe users password will be set to the password provided during deployment. 
+2. Download the latest Minecraft PE software from the Mojang website to ~minecraftpe (/home/minecraftpe)
+3. Append information from the user to the Minecraft PE server.properties configuration file
+4. Starts the Minecraft PE software as the minecraftpe user. 
+
+#### What Firewall ports are required to be opened?
+* 2x firewall ports need to be open in order to play that game over the internet: Port 19132 using TCP and Port 19132 using UDP
+
+#### How do I start the Minecraft PE server application?
+*. Login via ssh as the minecraftpe user and run the command:  screen ./start.sh
```
Comment (osbornm) [inline Ecosystem Partners/getting-started-with-minecraft-pe-server-blueprint.md]: did you mean for this to be `*. ` or is it suppose to be `* `

Comment (osbornm) [inline Ecosystem Partners/getting-started-with-minecraft-pe-server-blueprint.md]: Also in general for things that you type in fi you wrap them with tick (`) characters it helps them stand out. 

example: type `./start.sh` into the command prompt

**Comment by osbornm** [comment]
Fix those couple of things and LMK and I'll merge it in and release it.

**Comment by BobStolzberg** [comment]
Cool - Thanks a lot for the guidance, Matt!!  I'll make the changes and let you know when ready as you suggested.

Thanks again,

Bob Stolzberg | Sr. Cloud Integration Engineer
CenturyLink
Mobile/Text: (314) 397-2610
E-Mail: bob@CTL.io
www.CenturylinkCloud.com

**Comment by osbornm** [comment]
Images in list or bullets need to have an empty line (hit enter twice) with 2 spaces (per indent aka sub lists would get 4 spaces) in front of the image.

```
1. some test

   //start image tag here
```

**Comment by nathanyoung** [comment]
stale

**Pull request closed without merging**